src/Provider/Google.php

```diff
@@ -122,7 +122,8 @@ public function getUserProfile()
         $userProfile->emailVerified = $data->get('verified') ? $userProfile->email : '';
 
         if ($data->filter('image')->exists('url')) {
-            $userProfile->photoURL = substr($data->filter('image')->get('url'), 0, -2) . 150;
+             $photoSize = $this->config->get('photo_size') ?: '150';
```

Comment:
can you fix indent? (use 4 spaces instead of 5)

Reply:
Sure! Sorry about that.